int8 is_drop
int8 is_leave
int16 image_x
int16 image_y

